src/main/java/com/slack/eithernet/Extensions.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.eithernet
+
+/** If [ApiResult.Success], returns the underlying [T] value. Otherwise, returns null. */
+public fun <T : Any, E : Any> ApiResult<T, E>.successOrNull(): T? =
+  when (this) {
+    is ApiResult.Success -> value
+    else -> null
+  }
+
+/** If [ApiResult.Success], returns the underlying [T] value. Otherwise, returns null. */
```

Comment:
"Otherwise, returns the result of the [defaultValue] function" or similar

Reply:
Wanna suggest it so I can accept on mobile

Reply:
```suggestion
/** If [ApiResult.Success], returns the underlying [T] value. Otherwise, returns the result of the [defaultValue] function. */
```